.github/workflows/release.yml

```diff
@@ -90,8 +90,17 @@ jobs:
     name: 📝 Comment on related issues and pull requests
     if: github.repository == 'remix-run/react-router' && needs.find_package_version.outputs.package_version != ''
     needs: [release, find_package_version]
-    uses: ./.github/workflows/release-comments.yml
-    with:
-      ref: refs/tags/react-router@${{ needs.find_package_version.outputs.package_version }}
-      package_version_to_follow: "react-router"
-      release_branch: ${{ github.ref_name }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: ⬇️ Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: 📝 Comment on issues
+        uses: mcansh/release-comment-action@v0.1.3
+        env:
+          DIRECTORY_TO_CHECK: "./packages"
+          PACKAGE_VERSION_TO_FOLLOW: "react-router"
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          GH_TOKEN: ${{ github.token }}
```

Comment:
What about adding this as input instead of ENV variables?

This way you can set defaults for at least `GITHUB_REPOSITORY` & `GH_TOKEN` & you don't need to pass them anymore as the user of the Action
You can see an example of having this for `GH_TOKEN` here:
https://github.com/actions/labeler/blob/7012d51fe062f0b1e26e224a3c9fb52598ee3302/action.yml#L5-L8